### PR TITLE
L3-258: adding bitnami cert-manager helm chart to the l3-flux infra.

### PR DESCRIPTION
**Pull request by n3op2**
# What

Adding bitnami's `cert-manager` to the current l3-repo so it can be used along ClusterIssuer for protecting the entire cluster rather than per service base

> next steps - add ClusterIssuer(ACME)

### CODE REVIEW

File: clusters/l3-sqnc/cert-manager/kustomization-config.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - release.yaml
+configMapGenerator:
+  - name: cert-manager-values
+    files:
+      - values-cert-manager.yaml=values.yaml
+configurations:
+  - kustomize-config.yaml
+generatorOptions:
+  disableNameSuffixHash: true
+
```
[the remark this answers is not in the document]
Comment (n3op2) [reply clusters/l3-sqnc/cert-manager/kustomization-config.yaml]: this should be updated as in order, with the `cert-manager` namespace

File: clusters/l3-sqnc/base/app-sync.yaml [hunk 2]
```diff
@@ -40,3 +40,17 @@ spec:
...
+  name: cert-manager-sync 
+  namespace: cert-manager 
+spec:
+  interval: 1m
+  path: ./clusters/l3-sqnc/cert-manager
+  prune: true
+  sourceRef:
+    kind: GitRepository
+    name: flux-system
+    namespace: flux-system
```
Comment (dblane-digicatapult) [inline clusters/l3-sqnc/base/app-sync.yaml:56]: Sort out line end here.

File: clusters/l3-sqnc/cert-manager/release.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      sourceRef:
+        kind: HelmRepository
+        name: bitnami
+  interval: 10m0s
+  # Default values
+  # https://github.com/bitnami/charts/blob/master/bitnami/cert-manager/values.yaml
+  valuesFrom:
+    - kind: ConfigMap
+      name: cert-manager-values
+      valuesKey: values-cert-manager.yaml
```
Comment (dblane-digicatapult) [inline clusters/l3-sqnc/cert-manager/release.yaml:24]: Sort out line end here.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: clusters/l3-sqnc/cert-manager/release.yaml (hunk 3 above)
Comment (n3op2) [reply clusters/l3-sqnc/cert-manager/release.yaml]: went through entire folder to make sure formatting is consistent across

File: clusters/l3-sqnc/base/app-sync.yaml (hunk 2 above)
Comment (dblane-digicatapult) [inline clusters/l3-sqnc/base/app-sync.yaml:56]: Please sort out this line ending.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**